### DISCUSSION TITLE
Cell height is not auto fit by default anymore

### DISCUSSION THREAD

**Discussion by toygar00** [exceljs/exceljs#2520] [Ideas & Proposals]
Due to the below change, by default, cell height does not auto fit.
Need to set "worksheet.properties.defaultRowHeight = 0;" in order to auto-fit again. 
Not sure if this was intended with the below change

https://github.com/exceljs/exceljs/commit/902d7e48593a0f5009aeaa3af4bcc7a15d8a12b9

**Comment by walidvb** [comment]
Hi, i'm not able to have the rows auto-height by setting the default setting to 0, is this meant to add this behaviour?
Thanks!

**Comment by mllrdev-miguelsobong** [comment]
up

**Comment by ShadabFaiz** [comment]
No solution till now. Its messes up the cells which have really long text and is wrapped
